.buildkite/scripts/e2e-pipeline/bootstrap.py

```diff
@@ -73,6 +83,28 @@ def __clone_integrations_repo(self) -> None:
                                  "https://github.com/elastic/integrations.git"],
                                 "Error occurred while cloning an integrations repo. Check logs for details.")
 
+    def __scan_for_unsupported_processors(self) -> None:
+        curr_dir = os.getcwd()
+        pipeline_definition_file_path = "integrations/packages/**/data_stream/**/elasticsearch/ingest_pipeline/*.yml"
+        files = glob.glob(os.path.join(curr_dir, pipeline_definition_file_path))
+        unsupported_processors: dict = {}   # {type: file}
```

Comment:
should this be a dictionary of arrays, so that we can get _all_ integrations that use an unsupported processor?

Reply:
Updated with [`1c46477` commit](https://github.com/elastic/logstash-filter-elastic_integration/pull/242/commits/1c464771ae8f6c0d67255fe005dc92d923dc1735)
```
Exception: Unsupported processors found: {'rename_s': ['/logstash-filter-elastic_integration/integrations/packages/1password/data_stream/audit_events/elasticsearch/ingest_pipeline/default.yml', '/logstash-filter-elastic_integration/integrations/packages/1password/data_stream/signin_attempts/elasticsearch/ingest_pipeline/default.yml'], 'json_s': ['/logstash-filter-elastic_integration/integrations/packages/1password/data_stream/audit_events/elasticsearch/ingest_pipeline/default.yml']}
```